lxc/cluster_group.go

```diff
@@ -595,3 +599,64 @@ func (c *cmdClusterGroupShow) Run(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+// Add.
+type cmdClusterGroupAdd struct {
+	global  *cmdGlobal
+	cluster *cmdCluster
+}
+
+func (c *cmdClusterGroupAdd) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = usage("add", i18n.G("[<remote>:]<member> <group>"))
+	cmd.Short = i18n.G("Add member to group")
+	cmd.Long = cli.FormatSection(i18n.G("Description"), i18n.G(
+		`Add a cluster member to a cluster group`))
+
+	cmd.RunE = c.Run
+
+	return cmd
+}
+
+func (c *cmdClusterGroupAdd) Run(cmd *cobra.Command, args []string) error {
+	// Quick checks.
+	exit, err := c.global.CheckArgs(cmd, args, 2, 2)
+	if exit {
+		return err
+	}
+
+	// Parse remote
+	resources, err := c.global.ParseServers(args[0])
+	if err != nil {
+		return err
+	}
+
+	resource := resources[0]
+
+	if resource.name == "" {
+		return fmt.Errorf(i18n.G("Missing cluster member name"))
+	}
+
+	// Add the cluster group
```

Comment:
```suggestion
	// Retrieve cluster member information.
```
Calling `GetClusterMember` doesn't add the cluster group.

Reply:
Hi @monstermunchkin, I have updated that comment and also updated the translation files as well. sorry for the hindsight.